### DISCUSSION TITLE
How to use 0.01 babysteps during sd-print?

### DISCUSSION THREAD

**Discussion by FroggieFrog** [synman/Ender-3-S1-Pro-Firmware#49] [Q&A]
I'm trying to implement babystepping with 0.01 steps , but only during a sd-print (live adjustment).
During the levelling procedure I still want to use 0.05 steps.

Is there a reason why you use 0.01 everywhere, but use 0.05 during sd-print?
(If your source were available I could maybe check that myself.)

**Comment by lloydhass** [comment]
Hi FroggieFrog, I just saw your message and had asked something similar too. It would be nice if we could have both 0.05 & 0.01 steps in auto levelling and live adjustment.
